package.json

```diff
@@ -11,7 +15,9 @@
     "test-integration": "sh scripts/tests/testIntegration.sh",
     "test-unit": "sh scripts/tests/testUnit.sh",
     "test": "npm run lint && npm run test-integration && npm run test-unit",
-    "generate-api-schema": "node utils/generateAPISchema.js"
+    "generate-api-schema": "node utils/generateAPISchema.js",
+    "requirements-check": "node check_node_version.js",
+    "postinstall": ":.\node_modules/npm/bin/npm install && npm run requirements-check"
```

Comment:
What's `:.\node_modules/npm/bin/npm install` doing? 🤔

Reply:
Are we trying to access npm post build through this script?

Reply:
@swarajpure @harshith-venkatesh Reverted all the changes and added a new code for the check as I encountered with much simpler way to add a node version check, Please review.